src/main/java/uk/gov/digital/ho/hocs/workflow/client/caseworkclient/CaseworkClient.java

```diff
@@ -40,6 +40,13 @@ public CreateCaseworkCaseResponse createCase(String caseDataType, Map<String, St
         return response;
     }
 
+    public MigrateCaseworkCaseResponse migrateCase(String caseDataType, Map<String, String> data, LocalDate dateReceived, UUID fromCaseUUID) {
+        MigrateCaseworkCaseRequest request = new MigrateCaseworkCaseRequest(caseDataType, data, dateReceived, fromCaseUUID);
+        MigrateCaseworkCaseResponse response = restHelper.post(serviceBaseURL, "/migrate", request, MigrateCaseworkCaseResponse.class);
```

Comment:
It might seem irrelevant, but it would be more consistent for the receiving api to have this URI: `/case/{fromCaseId}/migrate`